### PR TITLE
Commiting movement

### PR DESCRIPTION
**Pull request by DevMaheshwari2017**
just added player movement an made jump anim more sexy an refined using rigidbody2d

### CODE REVIEW

File: Assets/Scripts/playerController.cs [hunk 1]
```diff
@@ -7,51 +7,63 @@ public class playerController : MonoBehaviour
      public Animator Animator;
     [SerializeField]
     float Jump_Power;
+    [SerializeField]
```
Comment (SahayDarshan) [inline Assets/Scripts/playerController.cs:10]: Follow the C# naming convention when writing variables.

File: Assets/Scripts/playerController.cs [hunk 2]
```diff
@@ -7,51 +7,63 @@ public class playerController : MonoBehaviour
...
+    {
+        RB2d = gameObject.GetComponent<Rigidbody2D>();
+    }
+    void PlayerMovement(float Horizonatal)
+    {
+        Vector2 pos = transform.position;
+        pos.x += Horizonatal * Speed * Time.deltaTime;
+        transform.position = pos;
+    }
+    void PlayingAnimation(float Horizonatl) {
```
Comment (SahayDarshan) [inline Assets/Scripts/playerController.cs:24]: Write access modifiers when writing the method.

File: Assets/Scripts/playerController.cs (hunk 1 above)
Comment (DevMaheshwari2017) [reply Assets/Scripts/playerController.cs:10]: Okay sure

File: Assets/Scripts/playerController.cs (hunk 2 above)
Comment (DevMaheshwari2017) [reply Assets/Scripts/playerController.cs:24]: yeah sometimes I don't do that, sorry for that

**Pull request merged**